mappings/net/minecraft/obfuscate/DontObfuscate.mapping

```diff
@@ -0,0 +1,25 @@
+CLASS net/minecraft/obfuscate/DontObfuscate net/minecraft/obfuscate/DontObfuscate
+	COMMENT This annotation applies to some unobfuscated elements within the Minecraft
+	COMMENT source code.
+	COMMENT
+	COMMENT <p>Its behavior appears as follows:
+	COMMENT <ul>
+	COMMENT <li>The annotation itself is not obfuscated.</li>
+	COMMENT <li>If a class is annotated, it is not obfuscated.</li>
+	COMMENT <li>If a member is annotated, it and its containing class is not obfuscated,
+	COMMENT but other members in the same class may stay obfuscated.</li>
+	COMMENT </ul>
+	COMMENT
+	COMMENT <p>In addition, single-abstract-method interfaces used as lambda expressions
+	COMMENT have their single abstract method unobfuscated without this annotation.
+	COMMENT
+	COMMENT <p>This annotation is not {@link java.lang.annotation.Documented}, and hence
+	COMMENT will not appear in the generated javadoc for annotated elements.
+	COMMENT
+	COMMENT @see net.minecraft.client.Main#main(String[])
+	COMMENT @see net.minecraft.data.Main#main(String[])
+	COMMENT @see net.minecraft.server.Main#main(String[])
+	COMMENT @see net.minecraft.client.ClientBrandRetriever#getClientModName()
+	COMMENT @see net.minecraft.server.MinecraftServer#getServerModName()
+	COMMENT @see net.minecraft.util.CubicSampler.RgbFetcher
+	COMMENT @see net.minecraft.world.level.ColorResolver#getColor()
```

Comment:
Ah, that's why `ColorResolver` was unobfuscated... I wonder *why* they did this.

Reply:
maybe they just had the anno by accident. i mean, that debug anno is always on methods except for one bee goal class